packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.1]
+### Changed
+- Add dependencies `eth-query` and `babel-runtime` ([#1447](https://github.com/MetaMask/core/pull/1447))
+-
+### Fixed
+- Fix bug where `query` function failed to call built-in EthQuery methods ([#1447](https://github.com/MetaMask/core/pull/1447))
```

Comment:
Here's a suggestion on addressing the other comment I left:

```suggestion
  - These dependencies were added to improve internal types. This has no impact on users of the package.
```

Reply:
Oh good catch, thanks, yes that sounds good

Reply:
Okay, updated here: [dc603d9](https://github.com/MetaMask/core/pull/1470/commits/dc603d900afd2492f12aa148cd4a8d8c9a5c7d2c)